lisa/messages.py

```diff
@@ -168,6 +190,38 @@ class DiskPerformanceMessage(PerfMessage):
     randwrite_iops: Decimal = Decimal(0)
     randwrite_lat_usec: Decimal = Decimal(0)
 
+    # Sysbench FileIO params
+    read_io_per_sec: Decimal = Decimal(0)
+    write_io_per_sec: Decimal = Decimal(0)
+    read_mib_per_sec: Decimal = Decimal(0)
+    write_mib_per_sec: Decimal = Decimal(0)
+    fsyncs_per_sec: Decimal = Decimal(0)
+    file_fsync_all: str = ""
+    file_fsync_end: str = ""
+    total_file: int = 0
+    file_total_size_in_gb: int = 0
+    file_async_backlog: int = 0
+    file_fsync_freq: int = 0
+    file_merged_requests: int = 0
+    file_rw_ratio: float = 0
+    file_ops: str = ""
+    file_io_mode: str = ""
+    file_fsync_mode: str = ""
+
+
+@dataclass
+class MemoryPerformanceMessage(PerfMessage):
+    total_operations: Decimal = Decimal(0)
```

Comment:
`operations_per_sec` is easier to understand. And similar to other fields.

Reply:
Remove it now